packages/astro/test/astro-scripts.test.js

```diff
@@ -1,18 +1,16 @@
-/**
- * UNCOMMENT: add Vite external script support
 import { expect } from 'chai';
 import cheerio from 'cheerio';
 import path from 'path';
 import { loadFixture } from './test-utils.js';
 
-let fixture;
+describe('Hoisted scripts', () => {
+  let fixture;
 
-before(async () => {
-  fixture = await loadFixture({ projectRoot: './fixtures/astro-scripts/' });
-  await fixture.build();
-});
+  before(async () => {
```

Comment:
Minor point: I typically like moving these above the `describe` since they’re not really scoped, and will execute a tick sooner.

Reply:
I put them in the describe block because if you don't and they throw you have no context of what is failing (Mocha doesn't tell you what file is throwing). If they are within the describe block Mocha will correctly tell you the suite that is failing in the before.